packages/prisma/src/prisma/schema.prisma

```diff
@@ -735,12 +735,17 @@ model ParticipantAccount {
 model Participant {
   id String @id @default(uuid()) @db.Uuid
 
+  email          String? @unique
+  isEmailValid   Boolean @default(false)
```

Comment:
not yet used, as preparation for potential email validation for manual accounts (not LTI). for validated LTI, we could set to true automatically.